src/chip.tsx

```diff
@@ -76,9 +80,12 @@ const Chip: React.FC<ChipProps> = ({Icon, children, id, dataAttributes, active,
         const isInteractive = active !== undefined;
         return (
             <Box
-                className={classnames(styles.chipVariants[active ? 'active' : 'default'], {
-                    [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
-                })}
+                className={classnames(
+                    styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
+                    {
+                        [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
+                    }
+                )}
```

Comment:
just for my own info: why does not the chip have the inverse?

Reply:
the chips have the same styles over inverse/default